examples/process-persistence/src/main/resources/META-INF/orm.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is no longer needed, we should bring the orm's from https://github.com/IBM/bamoe/pull/99